src/server/businesses.json

```diff
@@ -3,6 +3,8 @@
       "id": 1,
       "name": "Starbucks",
       "description": "We sell overpriced coffee.",
+      "imageUrl": "url(./images/1.png)",
```

Comment:
here we should only store the url like `./images/1.png` and add the `url()` in the code